ci/compare_sizes/project_instance.py

```diff
@@ -31,44 +31,52 @@ def branch(self, branch):
     def _examples_dir(self):
         return os.path.join(self._root_dir, "examples")
 
-    def _build_artifact(self, artifact, cache):
-        client = docker.from_env()
-        tag = "latest-arm64" if platform.machine() == "ARM64" else "latest-amd64"
-        image = f"nearprotocol/contract-builder:{tag}"
-
-        client.containers.run(
-            image,
-            "./build.sh",
-            mounts=[
-                docker.types.Mount("/host", self._root_dir, type="bind"),
-                docker.types.Mount(
-                    "/usr/local/cargo/registry", cache.registry, type="bind"
-                ),
-                docker.types.Mount("/usr/local/cargo/git", cache.git, type="bind"),
-                docker.types.Mount("/target", cache.target, type="bind"),
+    def _build_artifact(self, artifact):
+        subprocess.run(
+            [
+                "cargo",
+                "near",
+                "build",
+                "reproducible-wasm",
+                "--no-locked",
             ],
-            working_dir=f"/host/examples/{artifact.name}",
-            cap_add=["SYS_PTRACE"],
-            security_opt=["seccomp=unconfined"],
-            remove=True,
-            user=os.getuid(),
-            environment={
+            cwd=os.path.join(artifact),
+            check=True,
+            env={
+                **os.environ,  # Include the existing environment variables
                 "RUSTFLAGS": "-C link-arg=-s",
-                "CARGO_TARGET_DIR": "/target",
+                "CARGO_TARGET_DIR": f"{self._root_dir}/target",
             },
```

Comment:
Please remove `env=`, as the builds will run in docker, isolated from host env. 
Not sure this `*.py` code will run, but overall it looks legit to me.